site/docs/components/stepped-tracker/examples.mdx

```diff
@@ -20,6 +20,20 @@ You can add a label to the `TrackerStep` using the `StepLabel` component.
 
   </LivePreview>
 
+  <LivePreview componentName="stepped-tracker" exampleName="StageAndStatus" >
+
+## Stage and Status
+
+`SteppedTracker` supports multiple `stage` and `status` values for each `TrackerStep` child component.
+
+Available `stage` values are `"pending"` and `"completed"`. _Note that `"active"` is not a valid value: the active step is defined by the `activeStep` prop on the `SteppedTracker` component._
```

Comment:
The note should align to other notes in mdx e.g. _Note:_ `"active"` is not a valid value...

Reply:
looks like `**Note:** blah blah...` is the dominant format